mkosi/__init__.py

```diff
@@ -4559,6 +4555,14 @@ def run_verb(args: Args, images: Sequence[Config], *, resources: Path) -> None:
 
         check_workspace_directory(config)
 
+    if tools and not (tools.output_dir_or_cwd() / tools.output).exists():
+        if args.verb == Verb.build or args.force > 0:
+            fork_and_wait(run_sync, args, tools, resources=resources)
+            fork_and_wait(run_build, args, tools, resources=resources)
+        else:
+            die(f"Default tools tree requested for image '{last.name()}' but it has not been built yet",
+                hint="Make sure to build the image first with 'mkosi build' or use --force")
```

Comment:
I think it would make sense to put `--force` in single quotes, too, for consistency with `'mkosi build'`